frontend/src/components/TransferHistory.vue

```diff
@@ -38,7 +38,7 @@
     <div class="absolute bottom-0 right-0 w-full h-10 z-10 bg-gradient-to-t from-teal" />
   </div>
 
-  <SafeTeleport to="#action-button-portal">
+  <SafeTeleport v-if="signer" to="#action-button-portal">
```

Comment:
This was also not clear to me why necessary. I removed it locally to try it. Switching to the history while the selector is open would not show the new transfer button without this line. This also looks pretty unpredictable. Not sure why it works. Maybe because the if is reevaluated when you switch and the button is appended last in this case?

This seems to me like an example that the approach could be prone to errors in the future. What happens for example if we add another tab or any other kind of view?